### PR TITLE
CTPAT schema updates based on feedback

### PR DESCRIPTION
**Pull request by nissimsan**
This PR: 
- Introduces a CTPATMember RDF class because of requirements to allow multiple addresses and SCAC code on the organization
- Restricts the parts of Organization not needed for CTPAT members
- Flattens the CTPAT-Member-Location for easier implementation

### CODE REVIEW

**Review by BenjaminMoe — APPROVED** [review]

**Review by mkhraisha — APPROVED** [review]

File: docs/openapi/components/schemas/common/CTPAT.yml [hunk 1]
```diff
@@ -16,13 +16,197 @@ properties:
...
+                  type: string
+                  $linkedData:
+                    term: name
+                    '@id': https://schema.org/name
+                streetAddress:
+                  title: Street Address
+                  description: >-
+                    The street address expressed as free form text. The street address is
+                    printed on paper as the first lines below the name. For example, the name
+                    of the street and the number in the street or the name of a building.
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/CTPAT.yml:143]:
```suggestion
                    printed on paper as the first lines below the name. For example, the name
                    of the street and the number on the street, or the name of a building.
```

File: docs/openapi/components/schemas/common/CTPATMember.yml [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+  logo: 
+    title: Logo
+    description: An associated logo.
+    type: string
+    $linkedData:
+      term: logo
+      '@id': https://schema.org/logo
+  location:
+    title: Location
+    description: The location of, for example, where an event is happening, where an organization is located, or where an action takes place.
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/CTPATMember.yml:84]:
```suggestion
    description: The location where, for example, an event is happening, an organization is located, or an action takes place.
```

File: docs/openapi/components/schemas/common/CTPATMember.yml [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+              type: string
+              $linkedData:
+                term: name
+                '@id': https://schema.org/name
+            streetAddress:
+              title: Street Address
+              description: >-
+                The street address expressed as free form text. The street address is
+                printed on paper as the first lines below the name. For example, the name
+                of the street and the number in the street or the name of a building.
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/CTPATMember.yml:127]:
```suggestion
                of the street and the number in the street, or the name of a building.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Approved, with the changes above

**Pull request merged**